tests/Http/SignupTest.php

```diff
@@ -90,9 +90,6 @@ public function testNotCreatingDuplicateSignups()
     {
         $signup = factory(Signup::class)->states('contentful')->create();
 
-        // Mock the Blink API call.
-        $this->mock(Blink::class)->shouldReceive('userSignup');
-
         $response = $this->withAccessToken($signup->northstar_id)->postJson('api/v3/signups', [
```

Comment:
just curious why you deleted here!

Reply:
We're not creating a signup, and so this method is never called. Mocking it is unnecessary, and potentially harmful since if we started sending `userSignup` events here due to a bug we'd never know it.

Reply:
ohhh right because the signup already exists since we are creating it in the model factory - got it!